app/src/main/java/org/fossasia/susi/ai/rest/clients/DeviceClient.java

```diff
@@ -0,0 +1,24 @@
+package org.fossasia.susi.ai.rest.clients;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class DeviceClient {
+    private static Retrofit retrofit = null;
+
+    /**
+     * Gets client.
+     *
+     * @return the client
+     */
+    public static Retrofit getClient() {
+        if (retrofit == null) {
+            String SUSI_SPEAKER_BASE_URL = "http://10.0.0.1:5000";
```

Comment:
This is still contant

Reply:
@iamareebjamal instead of storing should I simply use the URL?

Reply:
How do you know that this is going to be the URL always? This is a local URL. You don't know the URL for sure, neither the port

Reply:
https://github.com/fossasia/susi_linux/blob/master/access_point/server/README.md

@iamareebjamal this will be the URL always as it is the raspi server, please refer the above link

Reply:
@iamareebjamal by configuration ?

Reply:
Means a person can change the URL in build.gradle

Reply:
@iamareebjamal if we have to make a variable in build.gradle too then what is the difference in both implementations ?

Reply:
Because then the person opening the project immediately sees the config parameter to edit

Also, add this in the README about how to change the smart speaker URL